### PR TITLE
WELD-1936 Expose BeanBuilder in Weld builder

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: weld/src/main/java/org/jboss/weld/experimental/BeanBuilder.java [hunk 1]
```diff
@@ -128,7 +163,7 @@
      * @param stereotypes
      * @return self
      */
-    BeanBuilder<T> stereotypes(@SuppressWarnings("unchecked") Class<? extends Annotation>... stereotypes);
```
Comment (jharting) [inline weld/src/main/java/org/jboss/weld/experimental/BeanBuilder.java]: Was it intentional to remove the varargs method for stereotypes?

Comment (mkouba) [inline weld/src/main/java/org/jboss/weld/experimental/BeanBuilder.java]: Yes and no ;-). We can't annotate this method with `@SafeVarargs` (it's not final nor static) and this version produces warning. But we should probably give the method back and add some info to javadoc?

Comment (jharting) [inline weld/src/main/java/org/jboss/weld/experimental/BeanBuilder.java]: I see. Let's leave it this way then for now.

**Comment by jharting** [comment]
Merged, thanks!

**Pull request closed without merging**